src/Mod/Assembly/UtilsAssembly.py

```diff
@@ -1083,11 +1087,12 @@ def findPlacement(ref, ignoreVertex=False):
     return plc
 
 
-def get_element(shape_elements, index, sub):
-    if index - 1 < 0 or index - 1 >= len(shape_elements):
-        print(f"Joint Corrupted: Index of {sub} out of bound.")
+def get_element(shape, name):
+    element = shape.getElement(name)
+    if element is None:
+        print(f"Unable to find element.")
```

Comment:
```suggestion
        App.Console.PrintWarning(f"Unable to find element {name}.")
```
I'd suggest either removing the `print` statement, or using the FreeCAD methods, and mentioning which element cannot be found. The assembly and solver code seem to have quite a few of these errors/warnings: the error is mentioned, but not the object it applies to, which always lefts the user wondering.